lib/rules/no-chained-get.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'disallow chain of `cy.get()` calls',
+      recommended: false,
+      url: 'https://github.com/cypress-io/eslint-plugin-cypress/blob/master/docs/rules/no-chained-get.md',
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      unexpected: 'Avoid chaining multiple cy.get() calls.',
```

Comment:
```suggestion
      unexpected: 'Avoid chaining multiple cy.get() calls',
```

Remove final period for consistency with other error messages